### PR TITLE
Allow override of badges

### PR DESCRIPTION
**Pull request by magopian**
Follows up on #3189, fixes [#1564](https://github.com/datagouv/data.gouv.fr/issues/1564)

### CODE REVIEW

File: udata/core/badges/tests/test_model.py [hunk 1]
```diff
@@ -155,3 +156,21 @@ def test_create_disallow_unknown_badges(self):
         with self.assertRaises(db.ValidationError):
             fake = Fake.objects.create()
             fake.add_badge("unknown")
+
+
+class DatasetBadgeTest(DBTestMixin, TestCase):
```
Comment (maudetes) [inline udata/core/badges/tests/test_model.py:161]: Do you want to move this in test_dataset_model?

**Review by maudetes — APPROVED** [review]
Thank you! I think we're good :clap:

Don't forget the changelog ;)

File: udata/core/badges/tests/test_model.py (hunk 1 above)
Comment (magopian) [reply udata/core/badges/tests/test_model.py]: Fixed the badge tests ;) What do you think?

Comment (maudetes) [reply udata/core/badges/tests/test_model.py]: :ok_hand: looks good, thank you!

**Pull request merged**